keyboards/tada68/keymaps/abishalom/rules.mk

```diff
@@ -0,0 +1,18 @@
+# Build Options
```

Comment:
No changes required here now, but just a general "for your information":

A keymap-level `rules.mk` doesn't need to set all the rules available, because the rules here only override the rules that are in `keyboards/tada68/rules.mk`.

Reply:
This is fine, actually, and personally I prefer it.